packages/core/package.json

```diff
@@ -5,7 +5,8 @@
     "@code-pushup/models": "*",
     "@code-pushup/utils": "*",
     "@code-pushup/portal-client": "^0.1.2",
-    "chalk": "^5.3.0"
+    "chalk": "^5.3.0",
+    "zod": "^3.22.4"
```

Comment:
Why? Only `models` should use `zod` directly, I believe. 

You don't need `globalOptionsSchema`, just the `GlobalOptions` type. Yargs doesn't use Zod schemas.

Reply:
ATM zod is used to provide default values. Ill refactor to yargs and types only